package/MDAnalysis/core/topologyattrs.py

```diff
@@ -772,7 +771,7 @@ def get_segments(self, sg):
 
     @warn_if_not_unique
     @check_pbc_and_unwrap
-    def center_of_mass(group, pbc=None, compound='group', unwrap=False):
+    def center_of_mass(group, pbc=False, compound='group', unwrap=False):
         """Center of mass of (compounds of) the group.
 
         Computes the center of mass of :class:`Atoms<Atom>` in the group.
```

Comment:
Would it be worth adding in the parameter docstring that pbc defaults to `False` now?

Reply:
It will have always defaulted to False, unless someone had used flags, which seems unlikely.

Reply:
That's fair 🙃